cmd/stfileinfo/main.go

```diff
@@ -70,7 +71,7 @@ func main() {
 		if *standardBlocks || blockSize < protocol.BlockSize {
 			blockSize = protocol.BlockSize
 		}
-		bs, err := scanner.Blocks(fd, blockSize, fi.Size(), nil, true)
+		bs, err := scanner.Blocks(context.TODO(), fd, blockSize, fi.Size(), nil, true)
```

Comment:
I wonder if we could stuff some of the cruft (sizeHint, Counter, useWeakHash, Counter) into context.WithValue.
For Counter and sizeHint it mostly makes sense as these are optional